src/templates/common/relatedLinks/index.tsx

```diff
@@ -11,41 +9,40 @@ export const RelatedLinks = ({
     return null
   }
 
-  let link
-  const renderLink = (uri, title, summary) => (
+  const renderLink = (uri: string, title: string, summary?: string) => (
     <>
       <p className="vads-u-margin--0">
-        <strong>
-          <va-link href={uri} text={title} />
-        </strong>
+        <va-link disable-analytics href={uri} text={title} />
```

Comment:
why the disable-analytics?

Reply:
I was wondering the same thing and I just chased that down. We are currently disabling DST analytics on these links in Prod so that's correct. But what I'm not sure about is is we are correctly capturing the custom event. [This is the ticket from sitewide](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18501) about this work, I'm not sure I can see the custom event in that PR though, is this something we have @cvalarida ?

Reply:
Genuinely have no idea. It's like that in prod. 🤷